src/ui/views/LandingPages/Welcome/GoogleImport/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+
+import { AllSet, LandingComponents } from '@/ui/FRWComponent/LandingPages';
+import { useNavigation, PageSlider } from '@/ui/utils/landingPage';
+import { storage } from 'background/webapi';
+
+import DecryptWallet from './DecryptWallet';
+import GoogleAccounts from './GoogleAccounts';
+import RecoveryPassword from './RecoverPassword';
+import RecoveryPhrase from './RecoveryPhrase';
+
+interface AccountsState {
+  accounts: string[];
+}
+
+const GoogleImport = () => {
+  const location = useLocation<AccountsState>();
+  const history = useHistory();
+  const { activeIndex, direction, goNext, goBack } = useNavigation(4);
+  const [mnemonic, setMnemonic] = useState('');
+  const [accounts, setAccounts] = useState<string[]>([]);
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const loadTempPassword = useCallback(async () => {
+    const temp = await storage.get('tempPassword');
+    if (temp) {
+      setPassword(temp);
+    }
+  }, []);
+
+  const getGoogleAccounts = useCallback(async () => {
+    const users = location?.state?.accounts;
+    setAccounts(users);
+  }, [location?.state?.accounts]);
+
+  useEffect(() => {
+    getGoogleAccounts();
+    loadTempPassword();
+  }, [getGoogleAccounts, loadTempPassword]);
+
+  return (
+    <LandingComponents
```

Comment:
I think these steps should just be included in the the addressimport steps. I don't think we should have a separate route for this